.travis.yml

```diff
@@ -1,10 +1,45 @@
-language: python
-python: 2.7
-env:
-  - TOXENV=flake8
-install:
-  - pip install tox
-script:
-  - tox
-notifications:
-  email: kpennington@mozilla.com
+cache: pip
+
+sudo: required
+jobs:
+  include:
+    - stage:
+      language: python
+      python: 3.6
```

Comment:
Make sure to set a current python version for legacy, which should be 2.7.12. The default one is 2.7.6